runtime/gc_structs/ConstantPoolClassSlotIterator.hpp

```diff
@@ -44,31 +44,30 @@ class GC_ConstantPoolClassSlotIterator
 {
 
 	J9Object **_cpEntry;
-	U_32 _cpEntryCount;
-	U_32 _cpEntryTotal;
-	U_32 *_cpDescriptionSlots;
-	U_32 _cpDescription;
-	UDATA _cpDescriptionIndex;
+	uint32_t _cpEntryCount;
+	uint32_t _cpEntryTotal;
+	uint32_t *_cpDescriptionSlots;
+	uint32_t _cpDescription;
+	uintptr_t _cpDescriptionIndex;
 	
 public:
-	GC_ConstantPoolClassSlotIterator(J9Class *clazz) :
-		_cpEntry((J9Object **)J9_CP_FROM_CLASS(clazz)),
-		_cpEntryCount(clazz->romClass->ramConstantPoolCount)
+	GC_ConstantPoolClassSlotIterator(J9Class *clazz)
+		: _cpEntry((J9Object **)J9_CP_FROM_CLASS(clazz))
+		, _cpEntryCount(clazz->romClass->ramConstantPoolCount)
 	{
 		_cpEntryTotal = _cpEntryCount;
-		if(_cpEntryCount) {
-			_cpDescriptionSlots = SRP_PTR_GET(&clazz->romClass->cpShapeDescription, U_32 *);
+		if (_cpEntryCount) {
```

Comment:
_cpEntryCount is an int, should we do 0 != .... ? @dmitripivkine

Reply:
yes, agreed